CHANGELOG.md

```diff
@@ -2,6 +2,287 @@
 
 # Changelog
 
+## 1.2 (2018-07-14)
+    Vendor in lates containers/image
+    build-using-dockerfile: let -t include transports again
+    Block use of /proc/acpi and /proc/keys from inside containers
+    Fix handling of --registries-conf
+    Fix becoming a maintainer link
+    add optional CI test fo darwin
+    Don't pass a nil error to errors.Wrapf()
+    image filter test: use kubernetes/pause as a "since"
+    Add --cidfile option to from
+    vendor: update containers/storage
+    Contributors need to find the CONTRIBUTOR.md file easier
+    Add a --loglevel option to build-with-dockerfile
```

Comment:
Should this be build-using-dockerfile instead?

Reply:
Well these are the actual changelogs.